### ISSUE TITLE
Detect Accept queue button outside of queue lock loop

### ISSUE THREAD

**Issue by RichAmber** [TeamFightTacticsBots/Alune#72]
Is it possible to make a bot accept a match in a party? I tried to run a bot in a lobby with a friend to get the party xp bonus, and it only accepts the match if you are the lobby leader. I think it doesn't realize the que loop has started, so it didn't try to press the accept button.

**Comment by akshualy** [comment]
You are correct. At the moment we need to start the queue before we look for the accept button. This is mainly due to the reason that there is no good way to differentiate between being in the lobby and being in queue, at least visually in the way we check these things. It has been a pain point for me that I wanted to fix for a while now, but have not come up with a good solution yet.
